### ISSUE TITLE
Failed to generate collection from WSDL with `anySimpleType`

### ISSUE THREAD

**Issue by nariman** [a85/POSTMan-Chrome-Extension#10484]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

I've tried to generate a Postman collection from WSDL spec and encountered the same issue as described in #10179, but with different root cause.
After some unsuccessful attempts to simplify the spec, I've figured out the problem by running `wsdl-to-postman` converter directly, which produces error, when spec contains an element definition with type [`anySimpleType`](https://www.w3.org/TR/xmlschema11-2/#special-datatypes).
Some sort of support for this datatype is needed (and probably for `anyAtomicType` too).

### Steps To Reproduce

Issue can be reproduced with [this example specification](https://github.com/postmanlabs/wsdl-to-postman/blob/4e6c62963a090fb900ba48cf47aa8536c385e85b/test/data/validWSDLs11/2schemasValid.wsdl) after changing the following line:
https://github.com/postmanlabs/wsdl-to-postman/blob/4e6c62963a090fb900ba48cf47aa8536c385e85b/test/data/validWSDLs11/2schemasValid.wsdl#L21
to:
```xml
<xsd:element minOccurs="0" name="city" nillable="true" type="xsd:anySimpleType" />
```

Console
```
There was an error getting json schemas from xsd  TypeError: this.builtInTypeConverter[typeName] is not a function
    at NamespaceManager.getBuiltInType (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:20217:44)
    at NamespaceManager.getTypeReference (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:20319:21)
    at ConverterDraft07.handleElementLocal (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:17283:46)
    at ConverterDraft07.element (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:17474:21)
    at ConverterDraft07.process (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:16729:38)
    at BaseConversionVisitor.visit (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:21595:31)
    at DepthFirstTraversal.walk (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:18148:39)
    at DepthFirstTraversal.walk (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:18155:20)
    at DepthFirstTraversal.walk (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:18155:20)
    at DepthFirstTraversal.walk (/workspaces/wsdl-to-postman/assets/xsd2jsonschemafaker.js:18155:20)
```

### Environment Information

```markdown
- Operating System: Win10
- Platform Type: Web, but reproducible on Native app for Linux
- Postman Version: Version 9.3.1-211203-1700 for Web
```

**Comment by akshaydeo** [comment]
@nariman Thanks for notifying us. We are looking into this issue, and we will update as soon as we fix the issue 🙇

**Comment by VShingala** [comment]
@nariman This issue is now fixed with Postman app version > 9.13.0. Feel free to reopen issue if you are facing further issues.
